[["file:///home/jfvilladiego/proyectos_laravel/cotizaciones_vcb_laravel/vendor/symfony/translation/Exception/RuntimeException.php",{"_uri":"file:///home/jfvilladiego/proyectos_laravel/cotizaciones_vcb_laravel/vendor/symfony/translation/Exception/RuntimeException.php","_root":{"kind":4096,"name":"file:///home/jfvilladiego/proyectos_laravel/cotizaciones_vcb_laravel/vendor/symfony/translation/Exception/RuntimeException.php","location":{"uriHash":581854903,"range":{"start":{"line":0,"character":0},"end":{"line":21,"character":0}}},"children":[{"kind":512,"name":"Symfony\\Component\\Translation\\Exception","location":{"uriHash":581854903,"range":{"start":{"line":11,"character":0},"end":{"line":11,"character":50}}}},{"kind":1,"name":"Symfony\\Component\\Translation\\Exception\\RuntimeException","location":{"uriHash":581854903,"range":{"start":{"line":18,"character":0},"end":{"line":20,"character":1}}},"children":[],"associated":[{"kind":1,"name":"RuntimeException"},{"kind":2,"name":"Symfony\\Component\\Translation\\Exception\\ExceptionInterface"}],"doc":{"description":"Base RuntimeException for the Translation component.","type":""},"modifiers":0}]},"_hash":581854903}]]